Packs/FortiSIEM/IncidentFields/incidentfield-Destination_User.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@OfekMQmasters Either add a prefix to the field name and cliname (like the other fields) or use the common "username" field instead (under "CommonTypes" pack).

Reply:
Done